activityical.php

```diff
@@ -397,3 +397,15 @@ function activityical_civicrm_tokenValues(&$values, $cids, $job = NULL, $tokens
     }
   }
 }
+
+/**
+ * Implements hook_civicrm_merge().
+ */
+function activityical_civicrm_merge($type, &$data, $mainId = NULL, $otherId = NULL, $tables = NULL) {
+  // remove activityical table in merge automatically added from foreign key
+  if (!empty($data['civicrm_activityicalcontact']))
```

Comment:
@lucky091588 Please use curly braces on all `if` structures. Per [civicrm/drupal coding standards](https://www.drupal.org/docs/develop/standards/coding-standards#controlstruct): "Always use curly braces even in situations where they are technically optional." Thanks.